shared/libraries/native_streaming_protocol/include/native_streaming_protocol/native_streaming_server_handler.h

```diff
@@ -31,7 +31,8 @@ BEGIN_NAMESPACE_OPENDAQ_NATIVE_STREAMING_PROTOCOL
 using OnSignalSubscribedCallback = std::function<void(const SignalPtr& signal)>;
 using OnSignalUnsubscribedCallback = std::function<void(const SignalPtr& signal)>;
 
-using SetUpConfigProtocolServerCb = std::function<ProcessConfigProtocolPacketCb(SendConfigProtocolPacketCb cb)>;
+using ConfigServerCallbacks = std::pair<ProcessConfigProtocolPacketCb, OnPacketBufferReceivedCallback>;
```

Comment:
Would it make sense to use a struct for this?

Reply:
I would keep this return type as a pair for now since it's not widely used and the it is very simple. However, I expect this part might need modifications soon anyway, so these could also include switching to a struct to improve code readability.